db/version_set.cc

```diff
@@ -2776,6 +2776,11 @@ Status Version::MultiGetAsync(
   to_process.emplace_back(0);
 
   while (!to_process.empty()) {
+    // As we process a batch, it may get split into two. So reserve space for
```

Comment:
Maybe it's better to be more explicit about which method (in this case `ProcessBatch()`) may cause `batches` to be resized.